backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/SpecificationController.kt

```diff
@@ -59,19 +58,19 @@ class SpecificationController @Autowired constructor(
         specificationHandler.findAll().map { spec -> specificationDtoMapper.fromDomain(spec) }
 
     @GetMapping("/{specificationId}")
-    fun findSpecification(@PathVariable specificationId: UUID): SpecificationDto {
+    fun findSpecification(@PathVariable specificationId: String): SpecificationDto {
```

Comment:
A completetly different idea: "Stringly typed" is a bad pattern, because you don't have the type safety anymore - anything can be a string. Do you think a small value object "SpecificationId" that just wraps the string makes sense?